src/rustllvm/rustllvm.h

```diff
@@ -45,7 +45,16 @@
 #include "llvm-c/ExecutionEngine.h"
 #include "llvm-c/Object.h"
 
-#if LLVM_VERSION_MINOR >= 7
+#define LLVM_VERSION_GT_OR_EQ(major, minor) \
```

Comment:
Perhaps this could be `LLVM_VERSION_GE`?

Similar to `ParialOrd::ge` naming at least